modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-configure-the-image.adoc

```diff
@@ -0,0 +1,48 @@
+= Initial Preparation and Configuration of the {azure} Image
+This section covers initial preparation and configuration of the image on {azure}.
+
+.Procedure: Configuring your {productname} instance
+
+. Start by logging into your Azure account at Azure Portal.
+
+. In the Azure Portal, find the menu:[Virtual Machines] section. This can be done either through the **dashboard** or by using the **search bar**.
+
+. Select menu:[Create a resource] in the top left corner of the portal.
+
+. Choose menu:[Compute] and then select menu:[Virtual Machine].
+
+. You will be guided through the **Create a virtual machine** process.
+
+. Fill in the required details such as subscription, resource group, and VM name.
+
+. Choose a region for your VM. Ensure it complies with any geo-fencing policies you might have.
+
+. In the **Image** dropdown, select the **manager-server-4-3-ltd** or the **manager-server-4-3-llc** image for your VM. If the image is not listed, select **Browse all public and private images** to find the required image.
+ 
+ . Suggested sizes are selected by default for this VM. Configure optional features such as virtual CPUs, memory as required.
+
+. Set up an administrator account for the VM. For {sles}, this can be a username and SSH public key.
+
+. Ensure the VM is connected to the appropriate virtual network (VNet) and subnet, especially if it needs to communicate with other specific services like {productname}.
+For network requirements, see: xref:public-cloud-guide/payg/azure/payg-azure-server-setup.adoc[Azure Server Setup: Network Configuration]
+
+. Configure additional settings such as storage, monitoring, or any extensions you may need. The following partitions are created by default when initializing the {azure} image:
+
+* **200 GB** for the root partition
```

Comment:
I think this is just `100 GB`

Reply:
Should also be changed in the AWS specific section to `100GB` for the root partition.

Reply:
ok done here, too